test/e2e/basic/available.go

```diff
@@ -53,6 +67,14 @@ var _ = ginkgo.Describe("[basic][available] Special Resource Operator availabili
 		gomega.Expect(err).NotTo(gomega.HaveOccurred(), explain)
 	})
 
+	// Create the preamble. If the operator is deployed using OLM ClusterOperator doesnt exist. If the resource
+	// doesnt exist then SRO will create one, but this only happens when creating a SpecialResource instance. To
+	// force it we create the preamble. If ClusterOperator was already present then preamble creation is idempotent.
+	ginkgo.It("Create preamble to enforce clusteroperator presence", func() {
+		ginkgo.By("Creating preamble")
+		createPreamble(cs, cl)
+	})
+
```

Comment:
It would be more consistent with the other tests to have createPreamble(...) return the error from ioutil.ReadFile(...), and then use gomega.Expect... Unless there is some reason that I am missing to panic on the error. 
```suggestion
		err := createPreamble(cs, cl)
		gomega.Expect(err).NotTo(gomega.HaveOccurred(), explain)
	})

```

Otherwise looking good!

Reply:
Done. Guess I may open a different PR to change the rest of the tests, as they follow the panic pattern.